src/Pages/Admin/Admin.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import "./Admin.css";
+
+const Admin = () => {
+
+    const retailers = [{name:"json"},{name:"sachin"},{name:"Praveen pandya"},{name:"Hardik pandya"},{name:"Virat Kholi"}];
+
+
+    return (
+        <div className="admin container-fluid">
+        <h1 className="admin-heading">Welcome to Admin Portal</h1>
+
+         <table class="table table-bordered">
+                <thead>
+                    <tr>
+                    <th scope="col">Retailer Id</th>
+                    <th scope="col">Name of the Retailer</th>
+                    <th scope="col">View Profile</th>
+                    <th scope="col">Delete Profile</th>
+                    </tr>
+                </thead>
+                <tbody>
+                        {
+                            retailers.map((user,index)=>(
```

Comment:
Center align this text to the column i.e name, index, buttons need to appear at the center and not at left.